calculate_age/calculate.py

```diff
@@ -0,0 +1,76 @@
+import time
+
+from calendar import monthrange, isleap
+
+
+# this function to convert a valid number to number of month
+def convert_age_to_months(age):
+    localtime = time.localtime(time.time())
+
+    try:
+        if isinstance(age, bool): 
+            raise
```

Comment:
Should raise TypeError exception. You don't raise without having any exception.

Reply:
If you don't raise the specific exception, main loop can't catch exactly. Raise an unknown exception which will make you hard to debug code.